video_xblock/video_xblock.py

```diff
@@ -293,6 +299,32 @@ def get_download_video_url(self):
         Else return `False` which will hide "download video" button.
         """
         return self.download_video_allowed and self.get_player().download_video_url
+    
+    @staticmethod
+    def _get_statici18n_js_url():
+        """
+        Returns the Javascript translation file for the currently selected language, if any found by
```

Comment:
Please split the title and body of multiple doc-strings by the empty line and control doc-strings are limited by 80 characters.